llvm/lib/Transforms/Instrumentation/InstrProfiling.cpp

```diff
@@ -885,6 +889,29 @@ void InstrLowerer::lowerValueProfileInst(InstrProfValueProfileInst *Ind) {
   Ind->eraseFromParent();
 }
 
+GlobalVariable *InstrLowerer::getOrCreateBiasVar(StringRef VarName) {
+  auto *Bias = M.getGlobalVariable(VarName);
+  if (Bias)
+    return Bias;
```

Comment:
```suggestion
  if (auto *Bias = M.getGlobalVariable(VarName))
    return Bias;
```

Reply:
It makes the scope of `Bias` smaller and causes errors.

Do you think I should introduce another variable for `new GlobalVariable`? I don't think so, because the characteristics of `Bias` is same here and it is tied to the return value.

That said, `auto` is lazy a bit. I'll rewrite shortly.